### PR TITLE
Add babel-eslint and babel-jest

### PR DESCRIPTION
**Pull request by isacjunior**
Close this issue #79 and add `babel-jest `

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
     "@babel/core": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^23.4.2",
```
Comment (existentialism) [inline package.json]: Not sure we need to add this to dev deps?

Comment (isacjunior) [reply package.json]: Without `babel-jest` I see this error:

```
  ● writes when --write is passed

    expect(jest.fn()).toBeCalled()
    
    Expected mock function to have been called.

      27 | 
      28 |   await writePackageJSON({ write: true });
    > 29 |   expect(writeJsonFile).toBeCalled();
      30 |   writeJsonFile.mockReset();
      31 | 
      32 |   await writeBabelRC("./fixtures/babelrc.json5", { write: true });
```
Node v10.9.0

Comment (aulisius) [reply package.json]: @isacjunior, which version of `node` are you on? I use node10 and I don't see that error when doing `npm run test`. If you are using node <  8, do `npm run build` and then run the tests.

Comment (isacjunior) [reply package.json]: Node: v10.9.0

Comment (aulisius) [reply package.json]: Ok. So,
`jest --version` gives `v22.4.4`
`node --version` gives `v10.0.0`
`babel-jest` is not installed.

and when I do `npm run test` on the `master` branch, all the tests run successfully.

Comment (isacjunior) [reply package.json]: Note: When I installed my dependencies with `npm` this errors above not occurred, but with `yarn` this error remains.

**Comment by isacjunior** [comment]
Without `babel-jest`occurred error that I had mentioned. And the proposed this PR is adding `babel-jest` when upgrading to Babel 7. For this reason, this project needs too `babel-jest`.


```  ● writes when --write is passed
    expect(jest.fn()).toBeCalled()
    
    Expected mock function to have been called.
      27 | 
      28 |   await writePackageJSON({ write: true });
    > 29 |   expect(writeJsonFile).toBeCalled();
      30 |   writeJsonFile.mockReset();
      31 | 
      32 |   await writeBabelRC("./fixtures/babelrc.json5", { write: true });
```

**Comment by isacjunior** [comment]
I'll close this PR and open others separated for each case.

**Pull request closed without merging**